src/10_functions.py

```diff
@@ -1,12 +1,15 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
-# YOUR CODE HERE
-
 # Read a number from the keyboard
 num = input("Enter a number: ")
 num = int(num)
 
-# Print out "Even!" if the number is even. Otherwise print "Odd"
+def is_even(num): 
+    if(num % 2 == 0):
+        print("Even")
```

Comment:
being a stickler here but the spec says you need to return something from `is_even()`